### DISCUSSION TITLE
getStaticProps() in Nextjs is not displaying the page source

### DISCUSSION THREAD

**Discussion by inesriahi** [vercel/next.js#28526] [Help]
I am following a udemy course about ReactJS which includes a section about Nextjs. I am following the instructor step by step and I don't seem to be getting the same result as he gets.

This is a simple application which displays a list of details of some meetings. Currently, I am trying to use `getStaticProps()` so that the HTML page source is displayed as well.

My problem is that I can't see the page source even with using `getStaticProps()`.

This is the page I am working on:
```
import MeetupList from "../components/meetups/MeetupList";
const DUMMY_MEETUPS = [
  {
    id: "m1",
    title: "first meetup",
    image:
      "https://images.unsplash.com/photo-1533167649158-6d508895b680?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1189&q=80",
    address: "Some Address 5, 12345",
    description: "This is the forst meetup",
  },
  {
    id: "m2",
    title: "second meetup",
    image:
      "https://images.unsplash.com/photo-1533167649158-6d508895b680?ixid=MnwxMjA3fDB8MHxwaG90by1wYWdlfHx8fGVufDB8fHx8&ixlib=rb-1.2.1&auto=format&fit=crop&w=1189&q=80",
    address: "Some Address 6, 12345",
    description: "This is the second meetup",
  },
];

const HomePage = (props) => {
  return <MeetupList meetups={props.meetups} />;
};
```

This is the `getStaticProps` function:
```
export async function getStaticProps() {

  // fetch data from API (not fetching anything currently)
  return {
    props: {
      meetups: DUMMY_MEETUPS,
    },
    // revalidate: 1, // will be regenrated after this number of seconds
  };
}

export default HomePage;
```

This is `_app.js`:
```
import "../styles/globals.css";
import Layout from "../components/layout/Layout";

function MyApp({ Component, pageProps }) {
  return (
    <Layout>
      <Component {...pageProps} />
    </Layout>
  );
}

export default MyApp;

```

This is a screenshot for the page I am generating:
[![enter image description here][1]][1]

However, this is the page source (As you can see, no text from the displayed one is showing here!):

[![enter image description here][2]][2]

This is the question on stackoverflow
https://stackoverflow.com/questions/68933589/getstaticprops-in-nextjs-is-not-displaying-the-page-source

  [1]: https://i.stack.imgur.com/qCOV5.png
  [2]: https://i.stack.imgur.com/WMu8j.png

**Comment by wwwdepot** [comment]
Your page source screenshot looks like from a Create-React-App.

**Reply by delbaoliveira** [reply]
Indeed. @onshadi, is it possible you instrumented the project with `npx create-react-app` rather than `npx create-next-app` or copied the wrong file from the course?

You can tell if it’s a Next project in the dependencies of your `package.json` or alternatively, referring to @wwwdepot said, if you open the dev tools, you should see a `div` with `id="__next"` under the `body` tag.
